source/application/BioXAS/BioXAS.h

```diff
@@ -32,8 +32,8 @@ namespace BioXAS
 		xasDefault->setColumnHeaderDelimiter("");
 		xasDefault->setSectionHeader("");
 		xasDefault->setSectionHeaderIncluded(true);
-		xasDefault->setIncludeAllDataSources(true);
-		xasDefault->setFirstColumnOnly(true);
+		xasDefault->setIncludeAllDataSources(false);
+		//xasDefault->setFirstColumnOnly(true); // Doesn't mean anything if all data sources aren't included.
```

Comment:
Ha, that was a mental note to myself. Would it be a good idea to develop that 'setFirstColumnOnly' capability for the case where all data sources are NOT included?